src/WebJobs.Script/Workers/Rpc/RpcWorkerConstants.cs

```diff
@@ -81,5 +81,8 @@ public static class RpcWorkerConstants
         public const string DotNetCoreDebugEngine = ".NETCore";
         public const string DotNetFrameworkDebugEngine = ".NETFramework";
         public const string DotNetFramework = "Framework";
+
+        public const string WorkerIndexingEnabled = "WORKER_INDEXING_ENABLED";
+        public const string WorkerIndexingDisabled = "WORKER_INDEXING_DISABLED";
```

Comment:
should we rename this to `WORKER_INDEXING_DISABLED_APPS` to make it more clear what it's for?